src/assests/errorIcon.js

```diff
@@ -0,0 +1,20 @@
+const ErrorIcon = props => {
+    const { height, width, fillColor, strokeColor, strokeWidth, strokeLinecap, strokeLinejoin } = props;
+    
+    return <svg xmlns='http://www.w3.org/2000/svg' height={height} width={width} viewBox='0 0 24 24' >
+        <path fill={fillColor} stroke={strokeColor} strokeWidth={strokeWidth} strokeLinecap={strokeLinecap} 
```

Comment:
no need to get strokeLineCap and strokeLinejoin from props

Reply:
Done!